packages/eas-cli/schema/metadata-0.json

```diff
@@ -472,49 +472,74 @@
         "description": "Info provided to the App Store review team to help them understand how to use your app.",
         "required": [
           "firstName",
-          "lastName"
+          "lastName",
+          "email",
+          "phone"
         ],
         "defaultSnippets": [
           {
+            "label": "Basic",
+            "description": "Only define the contact details in case communcation is needed with the App Store review team.",
+            "body": {
+              "firstName": "$1",
+              "lastName": "$2",
+              "email": "$3",
+              "phone": "+$4"
+            }
+          },
+          {
+            "label": "Full",
+            "description": "Define the demo account credentials, additional info, and contact details for the App Store review team.",
             "body": {
               "firstName": "$1",
-              "lastName": "$2"
+              "lastName": "$2",
+              "email": "$3",
+              "phone": "+$4",
+              "demoUsername": "$5",
+              "demoPassword": "$6",
+              "demoRequired": true,
+              "notes": "$7"
             }
           }
         ],
         "properties": {
```

Comment:
All descriptions are pulled from: https://developer.apple.com/documentation/appstoreconnectapi/prerelease_versions_and_beta_testers/beta_app_review_detail with some small edits to not-mention beta releases.